watch_requirements.txt

```diff
@@ -0,0 +1,2 @@
+formic # only python2 compatibile
```

Comment:
According to [pep8](https://www.python.org/dev/peps/pep-0008/#inline-comments), two spaces should be separated by at least two spaces.